src/Atmos/Model/AtmosModel.jl

```diff
@@ -213,49 +302,38 @@ Constructor for `AtmosModel` (where `AtmosModel <: BalanceLaw`).
 """
 function AtmosModel{FT}(
     orientation::Orientation,
-    param_set::AbstractParameterSet;
+    physics::AtmosPhysics;
```

Comment:
Can we have the same order of args in the AtmosModel function and the structure? e.g. physics, problem, orientation here

Reply:
I don't think so because the physics/orientation are non-optional at this point, and the problem/source etc. here are the defaults for all configs (GCM, LES etc.).